verilog/CST/declaration.cc

```diff
@@ -225,4 +227,55 @@ const verible::SyntaxTreeNode* GetStructOrUnionOrEnumTypeFromDataDeclaration(
   return GetStructOrUnionOrEnumTypeFromInstantiationType(*instantiation_type);
 }
 
+verible::SymbolPtr ReshapeExtendedFunctionCallInDataDeclaration(
```

Comment:
Can you provide in a comment what the code excerpt affected by this function might look like?

Reply:
Not necessary, this method is not there anymore.